include/openssl/opensslconf.h.in

```diff
@@ -68,9 +68,7 @@ extern "C" {
  * still won't see them if the library has been built to disable deprecated
  * functions.
  */
-#if defined(OPENSSL_NO_DEPRECATED)
-# define DECLARE_DEPRECATED(f)
-#elif __GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ > 0)
+#if __GNUC__ > 3 || (__GNUC__ == 3 && __GNUC_MINOR__ > 0)
```

Comment:
There was report that this triggers -Wundef warning/error with clang-cl, i.e. clang which pretends to be an msc.

Reply:
Why? Referring undefined macro is specified, so it's not like it's an error. Other compilers simply concentrate on *code* warnings, not pre-propcessor.

Reply:
Just in case, it wasn't my intention to examine it thoroughly. I mean I'm not actually looking into this matter, the #if thing simply caught my eye...